telco-hub/install/openshift/agent-config.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This CR doesn't have the host configuration. Isn't that required to define the networking parameters of the nodes?

Reply:
The `hosts` configuration can be found in the equivalent field of the _install-config.yaml_ (under `platform.baremetal.hosts`). It can be moved to the _agent-config.yaml_ file if you think it's a better place.

Reply:
The ABI documentation demonstrates networking configuration as part of the AgentConfig CR. I think we should align as closely as possible with that to avoid confusion or future incompatibility:
https://docs.openshift.com/container-platform/4.17/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#agent-install-networking-static_preparing-to-install-with-agent-based-installer

Reply:
Moved the `hosts` field to the _agent-config.yaml_. I also verified that the ABI installation works after the change.